templates/ocp4.3_pxeboot.j2

```diff
@@ -0,0 +1,16 @@
+DEFAULT {{ inventory_hostname }}-deploy
+PROMPT 0
+TIMEOUT 50
+TOTALTIMEOUT 6000
+ONTIMEOUT {{ inventory_hostname }}-deploy
+
+LABEL local
+        MENU LABEL (local)
+        MENU DEFAULT
+        LOCALBOOT 0
+
+LABEL {{ inventory_hostname}}-deploy
+        MENU LABEL RHCOS {{ rhcos_version }} {{ arch }}- Kickstart for {{ inventory_hostname }}
+        KERNEL http://{{ http_server_ip }}/rhcos-{{ rhcos_version }}-{{ arch }}-installer-kernel-{{ arch }}
+        APPEND ip={{ host_ip }}:dhcp rd.neednet=1 initrd=http://{{ http_server_ip }}/rhcos-{{ rhcos_version }}-{{ arch }}-installer-initramfs.{{ arch }}.img console=tty0 console=ttyS0 coreos.inst=yes rd.md=1 rd.auto coreos.inst.install_dev={{ md-raid-dev }} coreos.inst.image_url=http://{{ http_server_ip }}/rhcos-{{ rhcos_version }}-{{ arch }}-metal.{{ arch }}.raw.gz coreos.inst.ignition_url=http://{{ http_server_ip }}/bootstrap.ign
```

Comment:
I also doubt we want to use :dhcp  if we need to declare static variable. From quick look (but again haven't read the doc myself -no time for that now- ) we should instead use the following method : 
```
ip=<ip>::<gateway>:<netmask>:<hostname>:<device>:none nameserver=<dns>
```
So if we have that in ansible inventory (that's the goal) it should probably be : 
```
ip={{ ip }}::{{ gateway }}:{{ netmask }}:{{ inventory_hostname }}:{{ pxe_bootdev }}:none nameserver={{ nameserver }}
```
For pxe bootdev, we start centos linux with `net.ifnames=0 biosdevname=0` so we know it's `eth0`. Don't know about RHCOS kernel though but we can adjust with correct name after our first test

WRT {{ md-raid-dev }}  , that's something we should have in inventory too, but probably worth documenting in playbook header too. We can also then enhance later with jinja2 logic to have or not raid1 and rd.md=1 or not (trivial to add later)

Reply:
Also, for coreos.inst.ignition_url=http://{{ http_server_ip }}/bootstrap.ign , I'd have instead a full url to .ign file, so that it can be tuned with a vhost/path/to/file.ign easily. Would you mind having that added too ?

Reply:
Think ive addressed this correctly in latest commits.